### ISSUE TITLE
State value to set for "hm-rpc.1.xxxxxxx.1.LEVEL" has value "10" greater than max "1.01"

### ISSUE THREAD

**Issue by DutchmanNL** [ioBroker/ioBroker.hm-rpc#468]
**Describe the bug**
Error messages ar thrown to the log as the max value is set to 1.01, but the level can have a range from 0-100%
Issue occurs on Homematic IP - RadiatorThermostat of type HMIP-eTRV

**Expected behavior**
Correct min/max value to reduce error messages in log

**Screenshots & Logfiles**
```State value to set for "hm-rpc.1.xxxxxxx.1.LEVEL" has value "10" greater than max "1.01"```

```
{
  "type": "state",
  "common": {
    "def": 0,
    "type": "number",
    "read": true,
    "write": true,
    "min": 0,
    "max": 1.01,
    "unit": "%",
    "role": "level",
    "name": "Radiator_Luca_Slaapkamer.LEVEL",
    "custom": {
      "influxdb.0": {
        "enabled": true,
        "changesOnly": true,
        "debounce": "",
        "retention": 0,
        "changesRelogInterval": "1800",
        "changesMinDelta": 0,
        "storageType": "",
        "aliasId": ""
      }
    }
  },
  "native": {
    "MIN": 0,
    "OPERATIONS": 7,
    "MAX": 1.01,
    "FLAGS": 1,
    "ID": "LEVEL",
    "TYPE": "FLOAT",
    "DEFAULT": 0,
    "UNIT": "100%"
  },
  "_id": "hm-rpc.1.000393C99B49B6.1.LEVEL",
  "acl": {
    "object": 1636,
    "state": 1636,
    "owner": "system.user.admin",
    "ownerGroup": "system.group.administrator"
  },
  "from": "system.adapter.admin.0",
  "user": "system.user.admin",
  "ts": 1624305131281
}
```

**Versions:**
 - hm-rpc version: 1.15.5
 - hm-rega version: 3.0.35
 - JS-Controller version: 4.0.9
 - Node version: 14.9
 - Operating system: Ubuntu 20.4
 - CCU model: RaspberryMatic
 - CCU firmware: 3.61.7.20220115

**Comment by foxriver76** [comment]
have you tried deleting the datapoint once, unit 100% dps should be created with a max of 100

**Comment by DutchmanNL** [comment]
> have you tried deleting the datapoint once, unit 100% dps should be created with a max of 100

Yes I did, the level state did only come back when I activated the object synchronize once.
result of the object after that is still the same:

```
    "write": true,
    "min": 0,
    "max": 1.01,
    "unit": "%"
```

but I also saw another interesting bejavour, see my logging : 
<img width="1600" alt="Schermafbeelding 2022-02-15 om 16 07 32" src="https://user-images.githubusercontent.com/7318445/154089688-448a8311-02e5-4c3e-a3b2-3371b7cf7f97.png">

this. value should have been 14% instead of 0.14, or all values should be in 0.0 - 1.0 range
From all values I have checked of the past 2 days (200+ entry) this happend only once.

**Comment by foxriver76** [comment]
Currently not seeing how this can happen code-wise, at last we check if it is UNIT: 100% then say max = 100

**Comment by DutchmanNL** [comment]
> Currently not seeing how this can happen code-wise, at last we check if it is UNIT: 100% then say max = 100

hmm than lets make this even more weird, please have a look how my current logging looks like ( checked my logging and didn't receive this error anymore sins 15-02).

but also.... the values are write now totally different :  
<img width="1574" alt="Schermafbeelding 2022-02-20 om 16 42 33" src="https://user-images.githubusercontent.com/7318445/154850844-df496957-7adf-4a67-9f22-831161f2fae0.png">

so no everything is back to 0-1 scale... I am confused :/ wil monitor what happens anything I can add or specifically monitor/test to assist finding the RC ?

**Comment by DutchmanNL** [comment]
just to add, this is my history logging of the 15th

as far I remember only thing that happens in between was an adapter restart

<img width="1593" alt="Schermafbeelding 2022-02-20 om 16 44 11" src="https://user-images.githubusercontent.com/7318445/154850940-99d9270a-b844-4c71-9464-31f5f2c8671b.png">

**Comment by DutchmanNL** [comment]
ok no i m getting totally confused, it looks like the way of value handling has changed see my history.
If I remember correctly, I also updated RaspberryMatic during that time-window so maybe here was an issue that these values were submitted incorrectly ?
<img width="1613" alt="Schermafbeelding 2022-02-20 om 16 51 01" src="https://user-images.githubusercontent.com/7318445/154851292-413fd804-0387-4c12-913a-45a88c9e10e2.png">

**Comment by DutchmanNL** [comment]
sorry, I checked my installation twice and the update of raspberry magic is longer ago my virtual machine with update raspberry match runs already for 15 days no so was updated on the 5th of februari and not anywhere inn the time-frame of the 15th were this issue did occur so I see no relation
<img width="645" alt="Schermafbeelding 2022-02-20 om 16 53 13" src="https://user-images.githubusercontent.com/7318445/154851395-50eef26a-f161-4026-beef-94206dfee399.png">

**Comment by DutchmanNL** [comment]
Still relevant

**Comment by Kev0123** [comment]
I got the same issue 
![image](https://user-images.githubusercontent.com/19821853/180728014-d7b0f844-5b5e-43c1-8b56-bdbda06e551a.png)
